x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in transactions from your APM services.",
+  "description": "Detect anomalies in transactions from your APM services for metric data.",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*-transaction",
+  "defaultIndexPattern": "apm-*-metric,metrics-apm*",
   "query": {
     "bool": {
       "filter": [
-        { "term": { "processor.event": "transaction" } },
-        { "exists": { "field": "transaction.duration" } }
+        { "term": { "processor.event": "metric" } },
+        { "term": { "metricset.name": "transaction" } }
       ]
     }
   },
   "jobs": [
     {
-      "id": "high_mean_transaction_duration",
-      "file": "high_mean_transaction_duration.json"
+      "id": "apm_metrics",
```

Comment:
Shouldn't this be called `apm_transaction_metrics` since we have other types of metrics? We might want to add a separate job for system metrics in the future.

Reply:
I went with the more concise `apm_metrics` which is 19 characters shorter than the previous `high_mean_transaction_duration` to help address the problem of the APM job IDs potentially exceeding the 64 character limit for job IDs after APM prepends the environment and a random token, as discussed in https://github.com/elastic/kibana/issues/111037. I'll leave this one up to you @sqren - you could change this in a follow-up along with the detector description suggestion in https://github.com/elastic/kibana/pull/111957#pullrequestreview-769426809.